airbyte-cdk/bulk/toolkits/extract-trigger/src/main/kotlin/io/airbyte/cdk/TriggerCdcMetaFields.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk
+
+import io.airbyte.cdk.discover.CdcStringMetaFieldType
+import io.airbyte.cdk.discover.FieldType
+import io.airbyte.cdk.discover.MetaField
+import kotlin.text.lowercase
+
+/* We shouldn't have to create this but destination can't recognize this column if passed as field
+ * type. This is to be used as CURSOR_FIELD in TriggerTableConfig.
+ */
+enum class TriggerCdcMetaFields(
+    override val type: FieldType,
+) : MetaField {
+    CHANGE_TIME(CdcStringMetaFieldType);
```

Comment:
This represents a time I suppose.
Wouldn't `CdcOffsetDateTimeStringMetaFieldType` be better type here?

Reply:
It's fine btw to leave speed modifications to a seperate PR